nebula_ros/src/velodyne/hw_interface_wrapper.cpp

```diff
@@ -23,6 +25,11 @@ VelodyneHwInterfaceWrapper::VelodyneHwInterfaceWrapper(
       (std::stringstream{} << "Could not initialize HW interface: " << status_).str());
   }
 
+  if (!communicate_with_sensor_) {
+    // Do not initialize http client if communication is disabled
+    return;
```

Comment:
The `status_` is not set before returning, resulting in it being `NOT_INITIALIZED` instead of the expected `OK`.

Reply:
The `status_` should be set after sensor configuration is initialized ([here](https://github.com/msz-rai/nebula/blob/feature/disable-communication-option/nebula_ros/src/velodyne/hw_interface_wrapper.cpp#L20)).

Reply:
Ah, indeed. My bad!